macros/timeadd.sql

```diff
@@ -1,10 +1,11 @@
-{%- macro timeadd(part, amount_to_add, value) -%}
+{%- macro timeadd(part, amount_to_add, value, timestamp_cast_flag=true) -%}
     {#/* adds `amount_to_add` `part`s to `value`. so adding one hour to Jan 1 2020 01:00:00 would be timeadd('hour',1,'2020-01-01 01:00:00').
        NOTE: timeadd only manipulates time values. for date additions see [dateadd](#dateadd)
        ARGS:
          - part (string) one of 'second','minute','hour'.
          - amount_to_add (int) number of `part` units to add to `value`. Negative subtracts.
          - value (string) the date time string or column to add to.
```

Comment:
Can you adjust the type here to mention the support for datatime/timestamp type `(string or date/time type)` or something similar?

Also, you should be able to run the generate docs command to automatically update the documentation to include the type adjustment and new argument

Reply:
I've updated `docs/macros.md` by new description of `timeadd()`.